src/test/groovy/org/prebid/server/functional/tests/AuctionSpec.groovy

```diff
@@ -313,4 +314,36 @@ class AuctionSpec extends BaseSpec {
         and: "BidderRequest and bidRequest ids should be equal"
         assert bidderRequest.id == bidRequestId
     }
+
+    def "PBS should copy imp level passThrough to bidresponse.seatbid[].bid[].ext.prebid.passThrough when the imp.id matches seatbid[].bid[].impid"() {
+        given: "Default bid request with passThrough"
+        def textNode = TextNode.valueOf(PBSUtils.randomString)
```

Comment:
Pass-through is an object as far as I know. Am I wrong?

Reply:
Yes, you are right!